azure-pipelines.yml

```diff
@@ -87,6 +87,47 @@ stages:
       workingDirectory: $(Pipeline.Workspace)/
       displayName: 'Install swss-common dependencies'
 
+
+    - task: DownloadPipelineArtifact@2
+        inputs:
+          source: specific
+          project: build
+          pipeline: Azure.sonic-buildimage.common_libs
+          runVersion: 'latestFromBranch'
+          runBranch: 'refs/heads/master'
+          path: $(Build.ArtifactStagingDirectory)/download
+          ${{ if eq(parameters.arch, 'amd64') }}:
+            artifact: common-lib
+          ${{ else }}:
+            artifact: common-lib.${{ parameters.arch }}
+          patterns: |
+            target/debs/${{ parameters.debian_version }}/libprotobuf*.deb
+
+    - script: |
+          sudo dpkg -i $( find download -type f -name '*.deb' )
+        workingDirectory: $(Build.ArtifactStagingDirectory)
+        displayName: "Install protobuf"
+
+    - task: DownloadPipelineArtifact@2
+        inputs:
+          source: specific
+          project: build
+          pipeline: sonic-net.sonic-dash-api
+          ${{ if eq(parameters.arch, 'amd64') }}:
+            artifact: sonic-dash-api
+          ${{ else }}:
+            artifact: sonic-dash-api.${{ parameters.arch }}
+          patterns: |
+            libdashapi*.deb
+        displayName: "Download dash api deb packages"
+
+    - script: |
+            set -xe
+            sudo apt-get update
+            sudo dpkg -i libdashapi_*.deb
```

Comment:
Will this also install the dependency that is libprotobuf?

Reply:
It is done in the above script location (Line 106)